src/previewFrame/elementHighlighterScript.js

```diff
@@ -0,0 +1,36 @@
+const elementHighlighterScript = `(${function() {
+  window.addEventListener('message', (message) => {
+    const data = JSON.parse(message.data);
```

Comment:
We should add a try/catch here—in theory there’s no guarantee that the message is JSON.

Reply:
Oops forgot to add! will add in another commit